manual.md

```diff
@@ -131,15 +133,14 @@ Once proposed, participants can vote for/against the proposal on-chain with thei
 
 **Submission triggers a 3-day voting period. If passed, the proposal is queued for 48hrs, then it can be executed.**
 
-## Delegating
+## Delegation
 
-On-chain voting is enabled by delegating voting rights to the address (or addresses) of the token holder's choice:
+To participate in off-chain and on-chain voting, you have to first delegate the voting weight of your RAD. This can be done via various governance management platforms (e.g. [Tally](https://www.tally.xyz/gov/radworks) or [Boardroom](https://boardroom.io/radworks/proposals). After connecting your wallet to one of these platforms, you can choose to delegate the voting weight held in that wallet to the address (or addresses) of the token holder's choice:
```

Comment:
```suggestion
To participate in off-chain and on-chain voting, you must first delegate the voting weight of your $RAD. This can be done via various governance management platforms (e.g. [Tally](https://www.tally.xyz/gov/radworks) or [Boardroom](https://boardroom.io/radworks/proposals). After connecting your wallet to one of these platforms, you can choose to delegate the voting weight held in that wallet to the address (or addresses) of the token holder's choice:
```